lib/index.js

```diff
@@ -226,27 +237,27 @@ function listTokens (conf) {
   }
 }
 
-function removeToken (tokenKey, conf) {
+function removeToken (tokenKey, opts) {
   validate('SO', arguments)
-  const target = url.resolve(conf.registry, `-/npm/v1/tokens/token/${tokenKey}`)
-  return fetchJSON(Object.assign({target: target, method: 'DELETE'}, conf))
+  const target = `/-/npm/v1/tokens/token/${tokenKey}`
+  return fetch(target, ProfileConfig(opts, {
+    method: 'DELETE'
+  })).then(res => {
+    res.body.resume()
+    return null
```

Comment:
This whole thing does feel awkward, but I suppose not caring about a response body is unusual enough that it's fine.

Reply:
I've added `opts.ignoreBody` to `n-r-f`, which I think will help this pattern suck a bit less in the future. I'd rather just do that than do some weird heuristics, tbh. There's already `DELETE` API calls, for example, that _return the old value in the body_, so.